liferay-tango/liferay-portal
# LPS-123845 Remove the frontend OData parser and use a backend parser instead

Motivation*
=======
What is the problem this work solves, including
[Link to story or ticket](https://issues.liferay.com/browse/LPS-XXXX)

Proposed Solution*
========
What I/we did to solve this problem

Steps to Verify*:
----------------
1. A setup step / beginning state
1. What to do next
1. Any other instructions
1. Expected behavior
1. Suggestions for testing

Change summary (optional):
---------------
* Tidy, well formulated commit message
* Another great commit message
* Something else I/we did

Screenshots (optional):
-----------------------
Show-n-tell images/animations here

Checklist (optional):
========
